src/_includes/checklist__task.njk

```diff
@@ -1,4 +1,4 @@
-{% set slugifiedTitle = task.title | slugify %}
+{% set slugifiedTitle = task.title | striptags(true) | slugify %}
```

Comment:
@ericwbailey @SaptakS 🔍 

nunjucks' built-in `striptags` handles the case I wrote the extra `slugify` filter for: the `code` tags in the checklist task names